antenna/ext/crashstorage_base.py

```diff
@@ -17,6 +19,25 @@ class Config:
     def __init__(self, config):
         self.config = config.with_options(self)
 
+    def _path_join(self, *paths):
```

Comment:
Mmm... With Socorro, "crashstorage" isn't necessarily a file-based storage. In theory, it could be anything and in fact we have a crash storage for Elasticsearch and one for Telemetry which puts things in different places. I'm on the fence about whether we want to keep that thinking for Antenna or whether it makes it harder to share the GCS code between Antenna and Socorro.

I think if it were me, I wouldn't move these methods to `CrashStorageBase` but instead either create a `FileCrashStorageMixin` or duplicate the code between the S3 and GCS classes.

Reply:
i'll go ahead and duplicate the code

Reply:
Oh poop. I thought I did the "start a review" button. I didn't intend to make these one-off comments. Sorry about that.

Reply:
oop, i'll stop pushing changes then >_<